tests/visitor_actions.test

```diff
@@ -95,18 +95,43 @@ class VisitorActionsTest extends DrupalWebTestCase {
 
     $this->assertTrue(in_array($action_name, array_keys($va_settings['actions'])));
 
-    // Now edit the action and specify '<front>' as the path.
-    $edit = array(
-      'pages' => '<front>',
+    // Create a node and an alias to the node.
+    $node = $this->drupalCreateNode();
+    $node_path = 'node/' . $node->nid;
+    $alias_path = array(
+      'source' => $node_path,
+      'alias' => 'my_node',
     );
-    $this->drupalPost("admin/structure/visitor_actions/manage/$action_name/edit", $edit, t('Save'));
-    $this->assertNoText('The selected pages must contain valid paths.');
-    // Now edit the action and specify an invalid path.
-    $edit = array(
-      'pages' => 'some invalid url',
+    path_save($alias_path);
+
```

Comment:
You'll noticed that I have only tested the direct match, alias, and wildcards here. I did not get to test language prefixes because it appears to normally require turning "path translation" sub-module on from "internationalization" module, and create languages, and config, etc. Seems to not worth the effort. If you think differently please let me know.